CMakeLists.txt

```diff
@@ -1,142 +1,183 @@
 cmake_minimum_required(VERSION 3.15)
 
-cmake_policy(SET CMP0048 NEW)
-
-# Set cmake policies
-
-# FILE GLOB_RECURSE calls should not follow symlinks by default
-if(POLICY CMP0009)
-  cmake_policy(SET CMP0009 NEW)
-endif()
-# Error on non-existent dependency in add_dependencies
-if(POLICY CMP0046)
-  cmake_policy(SET CMP0046 NEW)
-endif()
-# CMP0074: find_package uses PackageName_ROOT variables
-if(POLICY CMP0074)
-  cmake_policy(SET CMP0074 NEW)
-endif()
-
 file(STRINGS "VERSION" pVersion)
 
 project(
   NCEPLIBS
   VERSION ${pVersion}
   LANGUAGES C Fortran)
 
-set(CMAKE_FIND_PACKAGE_PREFER_CONFIG true)
-
+include(ExternalProject)
 include(GNUInstallDirs)
 
-# set default install path if not provided
-if(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
-  set(CMAKE_INSTALL_PREFIX
-      "${CMAKE_BINARY_DIR}/install"
-      CACHE PATH "default install path" FORCE)
-endif()
-
 if(NOT CMAKE_BUILD_TYPE MATCHES "^(Debug|Release|RelWithDebInfo|MinSizeRel)$")
   message(STATUS "Setting build type to 'Release' as none was specified.")
   set(CMAKE_BUILD_TYPE
-      "Release"
-      CACHE STRING "Choose the type of build." FORCE)
+    "Release"
+    CACHE STRING "Choose the type of build." FORCE)
   set_property(CACHE CMAKE_BUILD_TYPE PROPERTY STRINGS "Debug" "Release"
-                                               "MinSizeRel" "RelWithDebInfo")
+    "MinSizeRel" "RelWithDebInfo")
 endif()
 
 if(NOT CMAKE_C_COMPILER_ID MATCHES "^(Intel|GNU|Clang|AppleClang)$")
   message(WARNING "Compiler not officially supported: ${CMAKE_C_COMPILER_ID}")
 endif()
 
-# When building on macOS using ifort add the -c flag to ranlib so that common symbols are
```

Comment:
Wondering whether this is still needed.

Reply:
I don't know, but I suspect we may need it on macOS.
I think we should first set-up some form of CI with different containers (macOS, Gentoo, etc.) and then add the missing bits where necessary, rather than blindly adding them from the beginning.

Reply:
@climbfuji I tried it on my macOS (GNU 9.3.0) and didn't need it.  I may need assistance on other machines.

Reply:
According to the comment (@kgerheiser when he added this change), it's needed for the Intel compiler on macOS, not GNU.

Reply:
I kinda recall this. I've got Intel so I can try it out.